ide/notifications.linux/src/org/netbeans/modules/notifications/linux/FallbackNotification.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.notifications.linux;
+
+import org.openide.awt.Notification;
+
+/**
+ *
+ * @author Hector Espert
+ */
+public class FallbackNotification extends Notification {
```

Comment:
The fallback should probably be the existing notification system, rather than a null implementation.

Reply:
Fallback is the existing notification system, `FallbackNotification` was the fallback of the fallback, but I removed this because it creates confusion.